site/content/entry/ldn-is-here/index.md

```diff
@@ -0,0 +1,200 @@
+Please respect the `Preferred Game` listed in publicly hosted rooms, as even unrelated games will add to the bandwidth load.
+
+{{< imgs "./room_chat.png| Public Room chat and moderation features" >}}
+
+<article class="message has-text-weight-semibold"><div class="message-body"><p>
+You can use these rooms for LAN mode games as well — instead of ZeroTier or Hamachi.</b>
+</p></div></article>
+
+## Development
+
+FearlessTobi began development by leveraging the existing virtual rooms infrastructure from [Citra](https://citra-emu.org).
+Due to code similarities between yuzu and Citra, this tried-and-tested infrastructure easily became the foundation of this feature.
+
+Now that the virtual rooms were taken care of, Tobi moved onto the `LDN` service and its protocols.
+Using the research and code from the [ldn_mitm](https://github.com/spacemeowx2/ldn_mitm) project, he implemented the necessary changes for yuzu instances to communicate with each other over these virtual rooms.
+While implementing, Tobi put an extra emphasis on ensuring no personal data like IPs was leaked to the other room members.
+
+As the feature slowly matured, Tobi began utilizing yuzu's internal team of testers to verify the feature across several titles for compatibility and performance validation. 
+
+With the multiple testing iterations, various bugs and glitches were found. 
+german77 relied on his own reverse-engineering and hardware tests to debug these issues, which made it easy for the team to fix them all.
+He also made several UI fixes to improve its usability and brought the codebase up to yuzu standards.
+
+## Compatibility
+
+Please note that as this is the initial release of the feature, some games may have some limitations.
+We hope to slowly fix these over the course of the next few months.
+
+yuzu's Local Wireless Multiplayer was tested by our team of Testers and Support staff across a small set of popular titles. Their compatibility findings are as follows:
+
+{{< imgs "./compat.png| List of yuzu-tested multiplayer titles" >}}
+
+
+## Screenshots
+
+{{< single-title-imgs
+    "Animal Crossing: New Horizons"
+    "./acnh1.jpg"
+    "./acnh2.png"
+>}}
+
+{{< single-title-imgs
+    "Pokémon Legends: Arceus"
+    "./arceustrade1.png"
+    "./arceustrade2.png"
+	"./arceustrade3.png"
+>}}
+
+{{< single-title-imgs
+    "Mario Kart 8 Deluxe"
+    "./mk8d_2.png"
+	"./mk8d_3.png"
+>}}
+
+{{< single-title-imgs
+	"Super Mario Smash Bros. Ultimate"
+	"./ssbu_ldn_3.png"
+	"./ssbu_ldn_4.png"
+	"./smash_ldn.png"
+>}}
+
+{{< single-title-imgs
+	"Splatoon 2"
+	"./splatoon2_1.png"
+	"./splatoon2_2.png"
+	"./splatoon2_3.png"
+>}}
+
+{{< single-title-imgs
+	"Luigi's Mansion 3"
+	"./lm3.png"
+	"./lm3_2.png"
+>}}
+
+{{< single-title-imgs
+	"Pokémon Let's Go: Pikachu/Eevee"
+	"./PLG_1.png"
+	"./PLG_2.png"
+>}}
+{{< single-title-imgs
+	"Pokémon Let's Go: Pikachu/Eevee"
+	"./PLG_3.png"
+	"./PLG_4.png"
+>}}
```

Comment:
```suggestion
{{< single-title-imgs
	"./PLG_1.png"
	"./PLG_2.png"
>}}
{{< single-title-imgs
	"Pokémon Let's Go: Pikachu/Eevee"
	"./PLG_3.png"
	"./PLG_4.png"
>}}
```
Remove redundant text from first set of images, as the 2nd subtitle can speak for all 4 images